src/EasySVG.php

```diff
@@ -240,6 +260,8 @@ public function textDef($text) {
         $fontSize = floatval($this->font->size) / $this->font->unitsPerEm;
         $text = $this->_utf8ToUnicode($text);
 
+        $prev_letter = '';
```

Comment:
for code consistency sake, would you mind using camel case for this variable?

I know it isn't standard in PHP but it seems better at this point to be consistent with the rest of the code